TESTING.md

```diff
@@ -324,6 +324,7 @@ While running tests, the following environment variables can be used:
     * `log` : writes full task details in the log
     * `simple_show` : displays a summary line for the task on screen
     * `simple_log` : writes a summary line for the task in the log
+* `GOVCD_IGNORE_CLEANUP_FILE` Ignore the cleanup file if it is left behind after a test failure.
```

Comment:
For me as a user this sentence raises a question: why should I want to do that? Please amend the description, so it's clear what problem it is solving.

Reply:
Description amended

Reply:
Typo in "On case when"? I suggest to shorten that whole sentence to:
> This could be useful after running a single test, when we need to check how the test behaves with the resource still in place.

Reply:
fixed